requirements/rtd.txt

```diff
@@ -1,4 +1,10 @@
 
+# Special version for RTD.
+
 -r docs.txt
```

Comment:
this dosen't work, we should pull it and replace it with the base and dev

Reply:
Sure, have to merge to find out!